tests/scripts/t3000/run_t3000_perplexity_tests.sh

```diff
@@ -11,6 +11,10 @@ run_t3000_perplexity_tests() {
   # Falcon7B perplexity tests
   WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -n auto models/demos/falcon7b/tests/perplexity/test_perplexity_falcon.py --timeout=1500 ; fail+=$?
 
+  # Llama-70B perplexity tests
+  WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -n auto models/demos/t3000/llama2_70b/demo/eval.py::test_LlamaModel_demo[wormhole_b0-True-wikitext-128-128-greedy-tt-70b-80L-llama3] --timeout=7200 ; fail+=$?
```

Comment:
Would be good practice to invoke eval.py directly instead of calling individual functions inside the .py file
https://github.com/tenstorrent/tt-metal/issues/7586 (for reference)

This makes the pipeline overall less brittle/more flexible long term since we can move towards invoking pytests in directories in the future and not having to call each function in every .py file